index.bs

```diff
@@ -889,6 +792,102 @@ user consent to a specific transaction. The structure of these signatures is def
     [[#op-get-assertion]].
 </div>
 
+## Additional options for Attestation (dictionary <dfn dictionary>MakeCredentialOptions</dfn>) ## {#dictionary-makecredentialoptions}
```

Comment:
Picked up in https://github.com/w3c/webauthn/pull/399/commits/6dca154104884ed814f2b0ded74a959bdf28e3ef.